src/Compilers/Core/MSBuildTask/Vbc.cs

```diff
@@ -689,7 +686,7 @@ internal void ParseVBErrorOrWarning(string singleLine, MessageImportance message
                     string originalVBErrorString = originalVBError.Message;
 
                     int column = singleLine.IndexOf('~') + 1;
-                    int endParenthesisLocation = originalVBErrorString.IndexOf(')');
+                    int endParenthesisLocation = originalVBErrorString.IndexOf(") :");
```

Comment:
```suggestion
                    int endParenthesisLocation = originalVBErrorString.IndexOf(") :", StringComparison.Ordinal);
```

As long as we are changing the code lets make the comparison explicit to avoid the `net5` changes around string comparison.